Configurations/windows-makefile.tmpl

```diff
@@ -599,9 +599,7 @@ EOF
      }
      return <<"EOF"	if (!$disabled{makedepend});
 $obj$depext: $deps
-	\$(CC) $cflags /Zs /showIncludes $srcs 2>&1 \\
-	    | "\$(PERL)" "\$(SRCDIR)\\util\\postprocess-makedepend.pl" \\
-	                 "VC" "$obj$objext" > $obj$depext
+	\$(CC) $cflags /Zs /showIncludes $srcs 2>&1 > $obj$depext
```

Comment:
It really hurts that `cl` outputs *everything* on stdout.  Had it output the includes data to stdout and error messages to stderr, it would have been much easier to use the `/showIncludes` flag in the normal compilation command instead of having to do this as a separate command...